flink-ai-flow/ai_flow/java/client/src/test/java/com/aiflow/client/AIFlowClientTest.java

```diff
@@ -0,0 +1,779 @@
+    }
+
+    @Test
+    public void testDeleteModelVersion() throws Exception {
+        String modelName = "test_delete_model_version";
+        String modelDesc = "test delete model version";
+        client.createRegisteredModel(modelName, modelDesc);
+
+        String modelPath = "fs://source1.pkl";
+        String modelType = "{\"flavor.version\":1}";
+        String versionDesc = "test delete model version1";
+
+        ModelVersion modelVersion = client.createModelVersion(modelName, modelPath, modelType, versionDesc);
+        Assertions.assertEquals("1", modelVersion.getModelVersion());
+
+        ModelVersion detail = client.getModelVersionDetail(modelName, "1");
+        Assertions.assertEquals(modelName, detail.getModelName());
+        Assertions.assertEquals(modelPath, detail.getModelPath());
+        Assertions.assertEquals("1", detail.getModelVersion());
+        client.deleteModelVersion(modelName, "1");
+        Assertions.assertNull(client.getModelVersionDetail(modelName, "1"));
+    }
+
+    // test notification service
+
+    @Test
+    public void testUpdateAndListNotification() throws Exception {
+        String namespace = "namespace";
+        String key = "send_event_key";
+        String value1 = "send_event_value1";
+        String eventType = "send_event_type";
+        String context = "send_event_context";
+        EventMeta event = client.sendEvent(namespace, key, value1, eventType, context);
+        Assertions.assertEquals(value1, event.getValue());
+        Assertions.assertTrue(event.getVersion() > 0);
+
+        List<EventMeta> eventList = client.listEvents(namespace, Arrays.asList(key), 0, eventType, 0);
+        Assertions.assertEquals(1, eventList.size());
+        Assertions.assertEquals(key, eventList.get(0).getKey());
+        Assertions.assertEquals(value1, eventList.get(0).getValue());
+        Assertions.assertEquals(eventType, eventList.get(0).getEventType());
+
+        String value2 = "send_event_value2";
+        EventMeta event2 = client.sendEvent(namespace, key, value2, eventType, context);
+        Assertions.assertEquals(event.getVersion() + 1, event2.getVersion());
+        eventList = client.listEvents(namespace, Arrays.asList(key), 0, eventType, 0);
+        Assertions.assertEquals(2, eventList.size());
+        Assertions.assertEquals(key, eventList.get(1).getKey());
+        Assertions.assertEquals(value2, eventList.get(1).getValue());
+        Assertions.assertEquals(eventType, eventList.get(1).getEventType());
+        eventList = client.listEvents(namespace, Arrays.asList(key), event.getVersion(), eventType, 0);
+        Assertions.assertEquals(1, eventList.size());
+        Assertions.assertEquals(key, eventList.get(0).getKey());
+        Assertions.assertEquals(value2, eventList.get(0).getValue());
+        Assertions.assertEquals(eventType, eventList.get(0).getEventType());
+    }
+
+    @Test
+    public void testListenNotification() throws Exception {
```

Comment:
It's recommended to remove this test case because this is tested in notifcation service.

Reply:
It is testing the notification methods in AIFlowClient, I think we can keep it.